src/Adapters.Tests/NLogTarget.Tests.Shared/NLogTargetTests.cs

```diff
@@ -126,6 +126,37 @@ public void TraceHasTimestamp()
 
         [TestMethod]
         [TestCategory("NLogTarget")]
+        public void TraceMessageCanBeFormedUsingLayout()
+        {
+            ApplicationInsightsTarget target = new ApplicationInsightsTarget();
+            target.Layout = @"${uppercase:${level}} ${message}";
+
+            Logger aiLogger = this.CreateTargetWithGivenInstrumentationKey("test", null, target);
```

Comment:
This is cool because if you want to switch to ILogger in future there is support for NLog and other frameworks already and you are also abstracting the logger instance acquisition. :+1: